### ISSUE TITLE
Vertipaq-Analyzer version 0.7.4

### ISSUE THREAD

**Issue by muhssamy** [microsoft/semantic-link-labs#153]
First of All, thanks for your continuous support.

i tested the new release and it works as expected. But i am missing the time stamp in which the vertipaq data has been exported to the lh tables so when the new data will come i will lose tracking as Example.

```python
spark.sql("DESCRIBE EXTENDED vertipaqanalyzer_tables").show(truncate=False)
``` 
```python
+-------------------+---------+-------+
|col_name           |data_type|comment|
+-------------------+---------+-------+
|Capacity_Name      |string   |null   |
|Capacity_Id        |string   |null   |
|Workspace_Name     |string   |null   |
|Workspace_Id       |string   |null   |
|Dataset_Name       |string   |null   |
|Dataset_Id         |string   |null   |
|Configured_By      |string   |null   |
|Table_Name         |string   |null   |
|Type               |string   |null   |
|Row_Count          |bigint   |null   |
|Total_Size         |bigint   |null   |
|Dictionary_Size    |bigint   |null   |
|Data_Size          |bigint   |null   |
|Hierarchy_Size     |bigint   |null   |
|Relationship_Size  |bigint   |null   |
|User_Hierarchy_Size|bigint   |null   |
|Partitions         |bigint   |null   |
|Columns            |bigint   |null   |
|%_DB               |double   |null   |
|                   |         |       |
+-------------------+---------+-------+
```

**Comment by m-kovalsky** [comment]
Thanks for reporting. Fixed in #154 which will be in the next release.

**Comment by m-kovalsky** [comment]
Fixed in [0.8.0](https://github.com/microsoft/semantic-link-labs/releases/tag/0.8.0).
